app/views/hub/documents/_form.html.erb

```diff
@@ -0,0 +1,20 @@
+<section class="slab" xmlns="http://www.w3.org/1999/html">
+  <div class="grid">
+    <%= form_with model: @document, url: url, local: true, method: method, builder: VitaMinFormBuilder do |f| %>
+
+      <h1 class="h2"><%= @title %></h1>
+      <div style="padding-bottom:2rem">
+        <%= f.label(:upload, t('general.select_files')) %>
+        <%= f.file_field :upload, multiple: "multiple", class: "attachment-upload file-input" %>
```

Comment:
This partial is used for edit and new, but the file upload field seems like it should only exist on new. Maybe we can use a variable on the partial to make sure that the file upload field only shows when rendering new and not edit!

Reply:
Got it!

Reply:
I considered that and I think there is a good use case for allowing users to update files for specific documents: downloading/rotating/cropping is a common use case.